.github/workflows/sanity-test.yaml

```diff
@@ -123,7 +123,7 @@ jobs:
             --hide-progress \
             --prefix $prefix \
             --bucket ${{ secrets.CI_TESTS_S3_BUCKET }} \
-            2>&1 | tee ./testlogs/gotest-repo-init.log
+            2>&1 | tee ./$TEST_RESULT/gotest-repo-init.log
```

Comment:
Will fix today, but for clarity: 

This pushes logs into two different packages: `./test_results/*` and `./testlogs/*`.  We should stick to one or the other, not both.  Going to keep with `testlogs` for consistency across gh actions.